src/components/NFTListElement.js

```diff
@@ -15,6 +15,15 @@ import {
 import { TOKEN_DOWNLOAD_STATUS } from '../sagas/tokens';
 import helpers from '../utils/helpers';
 import Loading from '../components/Loading';
+import { connect } from 'react-redux';
+import { numberUtils } from '@hathor/wallet-lib';
+
+
+const mapStateToProps = (state) => {
+  return {
+    decimalPlaces: state.serverInfo.decimalPlaces,
+  };
+};
```

Comment:
```suggestion
const mapStateToProps = (state) => ({
  decimalPlaces: state.serverInfo.decimalPlaces,
});
```

Reply:
This was not being used, so it was removed